shared/modules/parse_oval.py

```diff
@@ -25,16 +25,18 @@ class ElementFinder(object):
     def __init__(self, oval_groups):
         self.oval_groups = oval_groups
         self.target = None
+        self.attrib = None
         self.result = set()
 
-    def find_element(self, start_element, target_element_name):
+    def find_element(self, start_element, target_element_name, seeked_attrib):
```

Comment:
isn't this supposed to be "sought" or something like that? I dunno but seeked looks weird.

Reply:
Yeah,  you are probably right. I do not connect technical "seek" with the proper English word. Fixed ;)